package.json

```diff
@@ -1,37 +1,107 @@
 {
   "private": true,
-  "devDependencies": {
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/register": "^7.6.2",
-    "dotenv": "^8.1.0",
-    "eslint-plugin-compat": "^3.3.0",
-    "eslint-plugin-jasmine": "^2.10.1",
-    "globby": "^6.1.0",
-    "lerna": "^2.11.0"
+  "name": "@okta/okta-auth-js",
+  "description": "The Okta Auth SDK",
+  "version": "4.0.0",
+  "homepage": "https://github.com/okta/okta-auth-js",
+  "license": "Apache-2.0",
+  "main": "lib/server/serverIndex.js",
```

Comment:
should the `main` field point to built server bundle?

Reply:
Yes that is a good catch

Reply:
OK, actually this is correct. The package.json and the transpiled lib directory are built into the `dist` folder, which is where the module is published. So the path here is relative to the `dist` folder, not the repo root.